src/odemis/gui/model/__init__.py

```diff
@@ -1209,6 +1214,11 @@ def addStream(self, stream):
         If the stream is already present, nothing happens
         """
 
+        # TODO temporary, just testing
```

Comment:
Why is it temporary? Isn't it the the right behaviour? But it shouldn't look at the class, just whether the stream as a `.image` (if not => create a DataProjection).